golang/pkg/dagent/update/update.go

```diff
@@ -0,0 +1,183 @@
+package update
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/dyrector-io/dyrectorio/golang/internal/helper/docker"
+	"github.com/dyrector-io/dyrectorio/golang/internal/helper/image"
+	"github.com/dyrector-io/dyrectorio/golang/internal/util"
+	"github.com/dyrector-io/dyrectorio/golang/pkg/dagent/utils"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/client"
+
+	containerbuilder "github.com/dyrector-io/dyrectorio/golang/pkg/builder/container"
+)
+
+var _selfUpdateDeadline *int64
+
+func getContainerName(ctx context.Context, base string) (string, error) {
```

Comment:
Was this necessary?

Reply:
I want to generate a container name which is surely unique, but since most of the time I'm working with containerID-s I'm not sure 🤔